### PR TITLE
NewQuote

### PR DESCRIPTION
**Pull request by CQW-Code**
Signed-off-by: Christopher Watkin <chriswatkin@gmail.com>

# Pull Request

## Description:
Added two new quotes

Added two new quotes

## Related Issue:
  No issues and no issue # 

Closes #XXXXX

### CODE REVIEW

File: src/config/QuoteList.ts [hunk 1]
```diff
@@ -241,4 +241,13 @@ export const QuoteList: Quote[] = [
     text: "When I was young, I used to admire intelligent people; as I grow older, I admire kind people.",
     author: "Abraham Joshua Heschel",
   },
+  {
+    text: "Scars remind us where we've been. They don't have to dictate where we're going.",
+    author:  "David Rossi- Criminal Minds",
+
+  },
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:248]:
```suggestion
    author: "David Rossi- Criminal Minds",
  },
```

File: src/config/QuoteList.ts [hunk 2]
```diff
@@ -241,4 +241,13 @@ export const QuoteList: Quote[] = [
...
+  {
+    text: "Approach all areas of life with a bold enthusiasm this year",
+    author:  "A fortune cookie quote",
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:251]:
```suggestion
    author: "A fortune cookie quote",
```

File: src/config/QuoteList.ts [hunk 3]
```diff
@@ -241,4 +241,13 @@ export const QuoteList: Quote[] = [
...
+  }
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:252]:
```suggestion
  },
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Some linter fixes 🙂

**Comment by CQW-Code** [comment]
Thank you- will fix those in the next day or so- it's been a while since I've used everything (only VBA at work...a little :) )

I

**Comment by CQW-Code** [comment]
updated- I hope I did it right this time.   :)

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Pull request merged**